cloudinit/sources/helpers/openstack.py

```diff
@@ -648,7 +668,8 @@ def convert_net_json(network_json=None, known_macs=None):
 
             dns_nameservers = [
                 service["address"]
-                for service in network.get("services", [])
+                for route in network.get("routes", [])
```

Comment:
We are going to miss dns_nameservers from `services.address` if filled, shouldn't we add both?